### ISSUE TITLE
Add primary WCAG criteria to table on details page

### ISSUE THREAD

**Issue by MelSumner** [MelSumner/a11y-automation#157]
It would be useful to have the primary WCAG criteria on the data table on the details page, that way potential violations could be filtered/sorted on that column.

**Comment by legallai** [comment]
Hi, I'd like to help on this :)
